### PR TITLE
add coq.8.16.1

### PR DESCRIPTION
**Pull request by palmskog**
New release of Coq.

cc: @MSoegtropIMC @ppedrot

### CODE REVIEW

**Review by MSoegtropIMC — APPROVED** [review]
LGTM

**Comment by palmskog** [comment]
Only failure is a reverse dependency (frama-c 24.0).

File: packages/coq/coq.8.16.1/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+]
+depends: [
+  "ocaml" {>= "4.09.0"}
+  "ocamlfind" {>= "1.8.1"}
+  "dune" {>= "2.5.1"}
+  "conf-findutils" {build}
+  "zarith" {>= "1.11"}
+]
+conflicts: [
+  "base-nnp"
```
Comment (kit-ty-kate) [inline packages/coq/coq.8.16.1/opam:31]: Are there any news on the OCaml 5.0 support by any chance?

**Comment by kit-ty-kate** [comment]
Otherwise it looks fine. Thanks!

File: packages/coq/coq.8.16.1/opam (hunk 1 above)
Comment (palmskog) [reply packages/coq/coq.8.16.1/opam:31]: @kit-ty-kate no updates for a while to my knowledge, we still have two critical TODOs here: https://github.com/coq/coq/projects/55

Comment (kit-ty-kate) [reply packages/coq/coq.8.16.1/opam:31]: Is there any way I can help?

Comment (palmskog) [reply packages/coq/coq.8.16.1/opam:31]: I will pass on the offer of help to the Coq Zulip chat (there is an OCaml 5.0 topic there), and I guess we'll see what happens.

**Pull request merged**